frontend/src/components/Guesses.tsx

```diff
@@ -0,0 +1,19 @@
+import { ColoursForChoice } from '../models/ColoursForChoice'
+import "./Guesses.css"
+
+
+export const Guesses = () => {
+
+    let colourIndex : number[] = []
+
+    function inProgress(){}
```

Comment:
Was macht diese Funktion hier? Auch in ColourChoice.tsx hast du die so leer stehen